hypothesis-python/src/hypothesis/strategies/_internal/types.py

```diff
@@ -147,6 +147,38 @@
     pass  # `typing_extensions` might not be installed
 
 
+ReadOnlyTypes: tuple = ()
+try:
+    ReadOnlyTypes += (typing.ReadOnly,)  # type: ignore
+except AttributeError:  # pragma: no cover
+    pass  # Is missing for `python<3.12`
```

Comment:
Actually added in 3.13, per https://peps.python.org/pep-0705/

Reply:
Oups, the funny part is that I added it 😆